mrp_lot_cost/README.rst

```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+MRP Lot Unit Cost
+=================
+
+This module calculates the unit cost of a manufacturing lot taking the costs
+of buying lots of raw materials, the value is stored in stock.production.lot
+but this thought is not displayed as a tool for other modules
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/manufacture
```

Comment:
Thanks for your contribution
I think this is not necessary: I think it is a good idea but it should be consistent with other modules description.